instalooter/pages.py

```diff
@@ -42,12 +42,13 @@ class PageIterator(typing.Iterator[typing.Dict[typing.Text, typing.Any]]):
     _section_generic = NotImplemented    # type: Text
     _section_media = NotImplemented      # type: Text
 
-    def __init__(self, session, rhx):
+    def __init__(self, session, rhx, max_retry_count: int = 3):
```

Comment:
please decide on the default number based on some experiments to make sure that after `max_retry_count` we are really on an unfamiliar situation